# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-05-16 13:56+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: pinakes/common/image_mixin.py:43
msgid "{} object has no attribute 'icon'."
msgstr ""

#: pinakes/common/image_mixin.py:69
msgid "Icon attribute has been set on {} object (id: {})."
msgstr ""

#: pinakes/common/image_mixin.py:95 pinakes/common/image_mixin.py:123
msgid "Icon attribute has not been set on {} object (id: {})."
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:21
msgid "General platform configuration."
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:60
msgid "Data on sources"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:84
msgid "Data on service_offerings"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:114
msgid "Data on service_offering_nodes"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:145
msgid "Data on service_instances"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:175
msgid "Data on service_inventories"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:204
msgid "Data on portfolios"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:231
msgid "Data on portfolio_items"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:262
msgid "Data on orders"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:289
msgid "Data on order_items"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:323
msgid "Data on approval_requests"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:350
msgid "Data on service_plans"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:378
msgid "Data on templates"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:402
msgid "Data on workflows"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:429
msgid "Data on requests"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:464
msgid "Data on actions"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:490
msgid "Data on tag_links"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:517
msgid "Data on groups"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:539
msgid "Counts of orders by products"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:576
msgid "Counts of tags by portfolios"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:610
msgid "Counts of tags by products"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:644
msgid "Counts of tags by service_intentories"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:678
msgid "Order data by product"
msgstr ""

#: pinakes/main/analytics/analytics_collectors.py:777
msgid "Approval request time spent by group"
msgstr ""

#: pinakes/main/approval/exceptions.py:8
msgid "Insufficient query parameters"
msgstr ""

#: pinakes/main/approval/exceptions.py:13
msgid "Invalid State Transition"
msgstr ""

#: pinakes/main/approval/exceptions.py:18
msgid "Field cannot be blank"
msgstr ""

#: pinakes/main/approval/exceptions.py:23
msgid "Duplicated UUID for approver groups"
msgstr ""

#: pinakes/main/approval/exceptions.py:28
msgid "Group has no approver role"
msgstr ""

#: pinakes/main/approval/exceptions.py:33
msgid "Group does not exist"
msgstr ""

#: pinakes/main/approval/services/create_action.py:49
msgid "The memo message cannot be blank"
msgstr ""

#: pinakes/main/approval/services/create_action.py:56
msgid "Current request is not pending state"
msgstr ""

#: pinakes/main/approval/services/create_action.py:64
msgid "Current request is not started state"
msgstr ""

#: pinakes/main/approval/services/create_action.py:72
msgid "Current request is not in pending state"
msgstr ""

#: pinakes/main/approval/services/create_action.py:80
#: pinakes/main/approval/services/create_action.py:97
msgid "Current request is not in notified state"
msgstr ""

#: pinakes/main/approval/services/create_action.py:85
msgid "Only child level request can be approved"
msgstr ""

#: pinakes/main/approval/services/create_action.py:102
msgid "Only child level request can be denied"
msgstr ""

#: pinakes/main/approval/services/create_action.py:107
msgid "A reason has to be provided if a request is being denied"
msgstr ""

#: pinakes/main/approval/services/create_action.py:119
msgid "Only root level request can be canceled"
msgstr ""

#: pinakes/main/approval/services/create_action.py:124
msgid "The request has already finished"
msgstr ""

#: pinakes/main/approval/services/create_action.py:136
msgid "Current request has already finished"
msgstr ""

#: pinakes/main/approval/services/create_action.py:140
msgid "Failure reason is missing"
msgstr ""

#: pinakes/main/approval/services/create_action.py:144
msgid "Only child level request can be flagged error"
msgstr ""

#: pinakes/main/approval/services/email_notification.py:91
msgid "Failed to email group {}"
msgstr ""

#: pinakes/main/approval/services/update_request.py:19
msgid "Auto-approved"
msgstr ""

#: pinakes/main/approval/validations.py:56
msgid "Group id {} does not exist"
msgstr ""

#: pinakes/main/approval/validations.py:64
msgid "Group {} does not have approver role"
msgstr ""

#: pinakes/main/approval/views.py:189
msgid "Insufficient resource object params: {}"
msgstr ""

#: pinakes/main/catalog/exceptions.py:8
msgid "Bad query parameters"
msgstr ""

#: pinakes/main/catalog/exceptions.py:13
msgid "Invalid survey"
msgstr ""

#: pinakes/main/catalog/exceptions.py:18
msgid "Uncancelable Order"
msgstr ""

#: pinakes/main/catalog/models.py:537
msgid "Created Approval Request ref: {}. Catalog approval request id: {}"
msgstr ""

#: pinakes/main/catalog/services/cancel_order.py:61
msgid "Order {} is not cancel able in its current state: {}"
msgstr ""

#: pinakes/main/catalog/services/collect_tag_resources.py:62
msgid "Portfolio item {} does not have related service offering"
msgstr ""

#: pinakes/main/catalog/services/copy_portfolio_item.py:47
msgid "{} is not order able, and cannot be copied"
msgstr ""

#: pinakes/main/catalog/services/finish_order.py:32
msgid "Order {} has failed"
msgstr ""

#: pinakes/main/catalog/services/finish_order.py:37
msgid "Order {} is completed"
msgstr ""

#: pinakes/main/catalog/services/finish_order.py:54
msgid "This order item has failed due to the entire order {} before it ran"
msgstr ""

#: pinakes/main/catalog/services/finish_order_item.py:35
msgid "Order Item {} is completed"
msgstr ""

#: pinakes/main/catalog/services/handle_approval_events.py:106
msgid "Approval Request finished with status '{}'"
msgstr ""

#: pinakes/main/catalog/services/handle_approval_events.py:113
msgid "Approval Request finished with status '{}' and reason '{}'"
msgstr ""

#: pinakes/main/catalog/services/sanitize_parameters.py:42
msgid "parameter {} is required and must have a value"
msgstr ""

#: pinakes/main/catalog/services/start_order_item.py:54
msgid "Submitting Order Item {} for provisioning"
msgstr ""

#: pinakes/main/catalog/services/submit_approval_request.py:57
msgid "Failed to submit request to approval for Order {}, error: {}"
msgstr ""

#: pinakes/main/catalog/services/validate_order_item.py:45
msgid ""
"The underlying survey on product {} in the portfolio {} has been changed and "
"is no longer valid, please contact an administrator to fix it."
msgstr ""

#: pinakes/main/catalog/views.py:520
msgid "Order {} does not have related order items"
msgstr ""

#: pinakes/main/catalog/views.py:528
msgid "Computed tags for order {}: {}"
msgstr ""

#: pinakes/main/inventory/exceptions.py:8
msgid "Refresh already registered"
msgstr ""

#: pinakes/main/inventory/serializers.py:50
#, python-format
msgid "%(name)s is unavailable, refresh skipped"
msgstr ""

#: pinakes/main/inventory/serializers.py:55
#, python-format
msgid "Refresh failed: %(error)s"
msgstr ""

#: pinakes/main/inventory/serializers.py:59
msgid "adds"
msgstr ""

#: pinakes/main/inventory/serializers.py:60
msgid "deletes"
msgstr ""

#: pinakes/main/inventory/serializers.py:61
msgid "updates"
msgstr ""

#: pinakes/main/inventory/serializers.py:79
#, python-format
msgid "Service Inventories: %(stats)s;\n"
msgstr ""

#: pinakes/main/inventory/serializers.py:84
#, python-format
msgid "Job Templates & Workflows: %(soi_stats)s;\n"
msgstr ""

#: pinakes/main/inventory/serializers.py:89
#, python-format
msgid "Workflow Template Nodes: %(son_stats)s;\n"
msgstr ""

#: pinakes/main/inventory/serializers.py:93
msgid "Nothing to update"
msgstr ""

#: pinakes/main/inventory/serializers.py:116
#, python-format
msgid ""
"Source cannot be changed to url %(new_url)s uuid %(new_install_uuid)s, "
"currently bound to url %(url)s with uuid %(install_uuid)s"
msgstr ""

#: pinakes/main/inventory/serializers.py:122
#, python-format
msgid "Error: %(error)s"
msgstr ""

#: pinakes/main/inventory/services/collect_inventory_tags.py:38
msgid "ServiceOffering object {} not found"
msgstr ""

#: pinakes/main/inventory/services/get_service_offering.py:34
msgid "Failed to get service offering [{}]"
msgstr ""

#: pinakes/main/inventory/task_utils/check_source_availability.py:55
msgid "Check availability completed"
msgstr ""

#: pinakes/main/inventory/task_utils/launch_job.py:69
#, python-format
msgid "Job %(id)s has %(status)s status"
msgstr ""

#: pinakes/main/inventory/views.py:89
msgid "Refresh job {} is already {}, please try again later"
msgstr ""

#: pinakes/main/models.py:65
msgid "Done"
msgstr ""

#: pinakes/main/models.py:66
msgid "InProgress"
msgstr ""

#: pinakes/main/models.py:67
msgid "Failed"
msgstr ""

#: pinakes/main/models.py:68
msgid "Unknown"
msgstr ""
